core/utils.py

```diff
@@ -49,3 +51,12 @@ def get_approximate_date(date_str):
         except ValueError:
             return None
     return None
+
+def next_deadline():
+    """Return the next deadline for GitHub sponsoring so the organizers know when we will send their request."""
+    now = datetime.now()
+    start_date = datetime.strptime('04/09/16', '%d/%m/%y')
```

Comment:
And you can do the same thing in the tests too: you don't need to use `strptime` to create the date objects.